src/main/java/org/jitsi/jicofo/OctoChannelAllocator.java

```diff
@@ -81,30 +81,18 @@ public OctoChannelAllocator(
 
         boolean useIce = false;
```

Comment:
Is this even optional at this point?

Reply:
Yes, as you see Octo channels don't have ICE :)

Reply:
Ah